recipe/meta.yaml

```diff
@@ -40,6 +40,8 @@ about:
   license: GPL-3.0
```

Comment:
The `license` is `MPL-1.1 AND GPL-3.0-or-later AND LGPL-2.1-or-later`
see https://github.com/PyYoshi/cChardet/blob/master/COPYING

Reply:
Here is a confirmation https://github.com/PyYoshi/cChardet/blob/13076a3ab134c089b3460189a7d01d15b3f40f61/setup.py#L125-L127

Reply:
done